### DISCUSSION TITLE
BTraceUtil to handle array

### DISCUSSION THREAD

**Discussion by davidgargti20** [btraceio/btrace#552] [General]
Is there any BTrace util to handle array like iterating over it etc. ?

**Comment by jbachorik** [comment]
Not in the 'untrusted' mode.  The restrictions of the untrusted mode are primarily to prevent killing your app by eg. writing an endless loop by mistake.
You can still drop to 'trusted' mode where these restrictions are not applied and you can write standard for-loops (or anything else Java allows) to iterate over an array - and it is up to you to make sure that you are not going to kill your app (or it wouldn't matter if it is killed because it is eg. just a dev instance).
